docs/proposals/20200804-windows-support.md

```diff
@@ -131,7 +132,47 @@ and use privileged containers in place of wins.exe enabled containers.
 
 Each infrastructure providers must provide their own `PreKubeadmCommands`/`PostKubeadmCommands` scripts that
 are required for additional configuration for the node. During planning for Beta we will be able to identify
-common overlapping features that can be added into the the base images in image-builder and for re-use 
+common overlapping features that can be added into the the base images in image-builder for re-use.
+
+#### Kubeadm retry support 
+
+Based on the [CAPH Windows implementation](https://github.com/microsoft/cluster-api-provider-azurestackhci), it was 
+identified that the [experimental retry](https://github.com/kubernetes-sigs/cluster-api/pull/2763) for kubeadm 
+cluster provisioning (CABPK) will be beneficial for stability of Windows node joins.  The `UseExperimentalRetryJoin` 
+currently uses a Linux shell script to create add the retry functionality and does not work on Windows. The ideal 
```

Comment:
Can we just make this script into a cross-platform Go binary that we do the retry with?

Reply:
In the meeting there was a concern around versioning and binary distribution.  The end goal will be using a kubeadm as a library. 

The OsType information could be used by other providers as well which adds some additional value in providing the proposal above: https://github.com/kubernetes-sigs/cluster-api/pull/3616#discussion_r495132309

Reply:
Retry support in kube-adm was add in 1.19 and back-ported to 1.17.  As of the meeting today the goal is to drop the ExperimentalRetry via v1alpha. Any issues that arise we should open in kubeadm moving forward